src/style.css

```diff
@@ -2,0 +1,90 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+html {
+  font-family: 'Avenir';
+  font-weight: 300;
+}
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 20px;
+}
+
+.inputs__bigInput {
+  width: 100%;
+  font-family: 'Avenir';
+  font-size: 16px;
+  line-height: 150%;
+  height: 70px;
+  color: #3d4e61;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-sizing: border-box;
+  background-position: 25px;
+  padding: 0 0 0 62px;
+  outline: none;
+}
+
+.inputs__bigInput:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.inputs__bigInput:focus {
+  font-weight: 900;
+  box-shadow: 0 4px 4px 0 #00000040;
+  text-shadow: 0 4px 4px 0 #00000040;
+}
+
+.inputs__smallInput {
+  width: 100%;
+  font-family: 'Avenir';
```

Comment:
Consider adding a generic fallback font to the 'Avenir' font-family for the .inputs__smallInput class.